_observing-your-data/query-insights/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Query insights
+nav_order: 40
+has_children: true
+has_toc: false
+---
+
+# Query insights
+
+Query insights helps you monitor and analyze the search queries within your OpenSearch cluster. With minimal performance impact, query insights features aim to provide comprehensive insights into search query execution, enabling you to better understand search query characteristics, patterns, and system behavior during query execution stages. Query insights facilitate enhanced detection, diagnosis, and prevention of query performance issues, ultimately improving query processing performance, user experience, and overall system resilience.
+
+Typical use cases of query insights include the following:
+
+- Identifying top queries by latency within specific time frames.
+- Debugging slow search queries and latency spikes.
+
+Query insights features are backed up by the Query Insights plugin. At a high level, query insights comprises the following components:
+
+* _Collectors_: Gather performance-related data points at various stages of search query execution within OpenSearch.
+* _Processors_: Perform lightweight aggregation and processing on data collected by the collectors.
+* _Exporters_: Export the insights data into different sinks.
+
+
+## Installing the OpenSearch Query Insights plugin
+
+You need to install the `query-insights` plugin to enable query insights. To install the plugin, run the following command:
+
+```bash
+bin/opensearch-plugin install query-insights
+```
+For information about installing plugins, see [Installing plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/).
+
+## Query insights settings
```

Comment:
This heading is referenced in the settings documentation so it's better to have an explicit reference to query insights because then the link name is more precise.